### PR TITLE
Add commitment curve

### PR DESCRIPTION
**Pull request by jonallengriffin**
Initial inclusion of commitment curve.  I want to have this included in order to be able to create badges...I need to be able to point those badges at their criteria.

After badges are made, I'll update this with information on how to be nominated for the badges.

@wlach @jmaher please review.

### CODE REVIEW

**Comment by jmaher** [comment]
my only question is how will the badges be linked to this?  

Given as this is a medium for more description, should we have more details?  Personally I am not inclined to at this given moment.

File: guide/curve.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+Core Contributor
+----------------
+
+You've reached the point where you're a solid and consistent contributor to the A-Team.  You regularly participate in team meetings; file, review, and resolve bugs; and expand the number of Mozillians you work with.
+
+Criteria:
+
+* mozillians profile
+* 5+ good next bugs
```
Comment (wlach) [inline guide/curve.rst]: We talked about this a bit on Friday. How about we just say 5+ bugs here.

File: guide/curve.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+Explorer
+--------
+
+You're just beginning to investigate participation on the A-Team.
+
+Criteria:
+
+* Find bugs to work on
+* setup irc, bugzilla, hg/git, environment
```
Comment (wlach) [inline guide/curve.rst]: IRC should be capitalized, I think.

**Comment by wlach** [comment]
Can you capitalize the beginning of each bullet?

I'm worried that we're putting too much emphasis on good-first-bugs and good-next-bugs in this document, as opposed to smoothly integrating contributors into our process. In particular, I don't think we should have to tag good-next-bugs everywhere, it sounds like a lot of busywork... but this commitment curve sorta commits (heh) us to that. My experience is that after one or two good-next-bugs, contributors are pretty much ready to just work with us as equals.

Maybe we should define what good-first-bugs and good-next-bugs are somewhere in this document?

**Comment by jmaher** [comment]
I believe good first/next bugs provide a path for folks to get familiar
with their work and for us as mentors to get familiar with them.  the fact
of having a good*bug implies that we have done a bit more documenting than
normal, the bug has one or more mentors and newer contributors can be more
self starting without the fear of starting on an impossible bug.

With that said, we are spelling it out too formally in the criteria- I am
not sure of the best method forward - switch to mentored bugs?

On Mon, Jan 12, 2015 at 10:29 AM, William Lachance <notifications@github.com

**Comment by wlach** [comment]
Oh yes, I totally agree. The concept of good-*-bug is useful (we can debate the exact mechanics of that, but I think we agree in a general sense). I just don't want to tie the criteria for the more advanced levels to these types of bugs

**Comment by jonallengriffin** [comment]
> my only question is how will the badges be linked to this?
> 
> Given as this is a medium for more description, should we have more details? Personally I am not inclined to at this given moment.

The badge description will include something like "Earned after meeting the criteria specified at <link>."  I agree about not needing more details right now.

**Comment by jonallengriffin** [comment]
> The badge description will include something like "Earned after meeting the criteria specified at ."

Er, that should be "Earned after meeting the criteria specified at [link to doc in A-Team Bootcamp]."

**Comment by jmaher** [comment]
sold!

**Comment by jonallengriffin** [comment]
PR updated; the second tier now includes "Resolve 3+ good first bugs" plus "Resolve at least one mentored bug which isn't a good first bug".

File: guide/curve.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Core Contributor
+----------------
+
+You've reached the point where you're a solid and consistent contributor to the A-Team.  You regularly participate in team meetings; file, review, and resolve bugs; and expand the number of Mozillians you work with.
+
+Criteria:
+
+* Create a Mozillians profile
+* Resolve 5+ good next bugs
```
Comment (wlach) [inline guide/curve.rst]: Can we change this to 5+ bugs (not good first bugs)

Comment (jmaher) [inline guide/curve.rst]: that seems good!

**Pull request merged**